cpp/src/knn/knn_opg_common.cuh

```diff
@@ -576,7 +576,7 @@ void exchange_results(opg_knn_param<in_t, ind_t, dist_t, out_t>& params,
 
     if (params.knn_op != knn_operation::knn) {
       requests.resize(2 + params.n_outputs);
-      for (size_t o = 0; o < params.n_outputs; o++) {
+      for (int o = 0; o < params.n_outputs; o++) {
```

Comment:
It looks like we can safely change the type of `n_outputs` to allow us to correctly use a `std::size_t` for iteration here.

Reply:
Done.